src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const resolve = () => {
```

Comment:
Create a promise with the `resolve` function as the executor. The `resolve` function should not add an event listener, but instead directly resolve the promise.